openedx/core/djangolib/tests/test_markup.py

```diff
@@ -90,3 +90,10 @@ def test_strip_all_tags_but_br_filter(self):
 
         self.assertIn('<br>', rendered_template)
         self.assertNotIn('<script>', rendered_template)
+
+    def test_strip_all_tags_but_br_returns_html(self):
+        """ Verify filter returns HTML Markup safe string object"""
```

Comment:
Nit: Guidelines say this should be:
```
"""
Verify filter returns HTML Markup safe string object
"""
```
If you do put on one line, you should be consistent about adding a space before and after.